dydx/dydxFormatter/dydxFormatter.xcodeproj/xcshareddata/xcschemes/dydxFormatterTests.xcscheme

```diff
@@ -14,8 +14,7 @@
       shouldAutocreateTestPlan = "YES">
       <Testables>
          <TestableReference
-            skipped = "NO"
-            parallelizable = "YES">
```

Comment:
why get rid of parallelizable?

Reply:
When it's turned on, my xcode hangs when running tests.